modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {ColorInput} from '@workday/canvas-kit-react-color-picker';
+import {borderRadius, colors, spacing} from '@workday/canvas-kit-react-core';
+import {Label} from '@workday/canvas-kit-react-form-field';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import {checkIcon} from '@workday/canvas-system-icons-web';
+
+import {SwatchBook} from './parts/SwatchBook';
+import {ResetButton} from './parts/ColorReset';
+
+export interface ColorPickerProps {
+  ariaLabel?: string;
+
+  onChange: (color: string) => void;
```

Comment:
This signature brings up an open question I had. I created an issue for this: #487 

It is very easy to create a signature like this, but it has ramifications downstream to consumers of form fields.

Reply:
this current type signature also causes conflicts with https://github.com/Workday/canvas-kit/pull/462#discussion_r383600029

Reply:
also, how should I handle this for now?

Reply:
Good question. I think we can leave it for now. This is going into labs and the question is far-reaching and might not be answered quickly. I say leave it for now.

As for https://github.com/Workday/canvas-kit/pull/462#discussion_r383600029, you can extract `onClick` and pass `elemProps` (or rest props) to the container. The comment would require you to extend the `ColorPickerProps` interface from `React.DivHTMLAttributes<HTMLDivElement>`,